### ISSUE TITLE
A11y: extension license buttons and links lack context

### ISSUE THREAD

**Issue by JDGrimes** [WordPoints/wordpoints#694]
These buttons have context based on where they are on the page, but that doesn't help unsighted users. Just as for the update buttons/links, these need to be labeled with the extension title.
